src/DatePicker/DatePicker.js

```diff
@@ -284,15 +336,20 @@ class DatePicker extends Component {
     const formatDate = formatDateProp || this.formatDate;
 
     return (
-      <div className={className} style={prepareStyles(Object.assign({}, style))}>
+      <div ref={(node) => this.element = node} className={className} style={prepareStyles(Object.assign({}, style))}>
         <TextField
           {...other}
           onFocus={this.handleFocus}
           onTouchTap={this.handleTouchTap}
+          readOnly={true}
```

Comment:
Can we have it before the `{...other}` spread? We need users to be able to override it.
```js
<TextField
    readOnly={true}
    {...other}
```